js/app/Visualization/UI/SidePanels/SidePanelManager.js

```diff
@@ -45,7 +45,7 @@ define([
         '        <a id="feedback_url" target="_blank">' +
         '          Feedback' +
         '        </a>' +
-        '        <div id="collapse-button"><img src="%(img)s/buttons/close.png"></div>' +
+        '        <div id="collapse-button"><i class="fa fa-chevron-circle-right"></i><!--img src="%(img)s/buttons/close.png"--></div>' +
```

Comment:
@jarmstrongbts The following is just a personal opinion, I did not like how this new icon looks like in the menu. It's the most predominant icon when you give a quick look to the menu. What about using `chevron-right` with NO circle for closing?